lib/dhis2_cli/models/import/data_values_file.rb

```diff
@@ -0,0 +1,31 @@
+# Represent a data value file
+class DataValuesFile
+  attr_reader :filepath, :headers, :lines
+
+  def initialize(filepath, headers, lines)
+    @filepath = filepath
+    @headers = headers
+    @lines = lines
+  end
+
+  def data_element_headers
+    @data_element_headers ||= @headers.select { |header| header.start_with?('de') }
+  end
+
+  def data_values_lines
+    @data_values_lines ||= @lines.each_with_index.map do |line, idx|
```

Comment:
it's a rails method, but in ruby you can do arr.map.with_index for simplier syntax